scrapers/utils.py

```diff
@@ -123,8 +123,10 @@ def get_md5hash(filepath):
 def validate_and_get_download_link(base_url, download_link, check_content_pdf=True):
     if not download_link or not isinstance(download_link, str):
         raise ValueError("File URL is invalid.")
-
-    absolute_url = f"{base_url}/{download_link}"
+    if download_link.startswith("/"):
+        absolute_url = f"{base_url}{download_link}"
+    else:
+        absolute_url = f"{base_url}/{download_link}"
```

Comment:
Or alternatively, we can strip the first `/` in `download_link`? This may be more manageable and less duplicated code

```suggestion
    download_link = download_link.replace("/", 1) if download_link.startswith("/") else download_link
    absolute_url = f"{base_url}/{download_link}"
```

Reply:
made this change